test/parallel/test-net-listen-close-server-callback-is-not-function.js

```
@@ -0,0 +1,16 @@
+var common = require('../common');
+var assert = require('assert');
+var net = require('net');
+var gotError = false;
+
+var server = net.createServer(function(socket) {
+});
+server.listen(common.PORT, function() {
+  assert(false);
+});
+server.on('error', function(error) {
+  common.debug(error);
+  assert(false);
+});
```

If you're not handling the error, just let it bubble up.